src/pages/AllMoviesPage.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <h1>All Movies Page</h1>
+    <router-link to="/movie/1">Movie Number 1</router-link>
+    <a href="" @click="movie2">Movie Number 2</a>
```

Comment:
Have you considered to try and use names here to reference the routes by their unique names?

I really encourage that - especially in the future as it allows us to change the routes in our application much more easy - if we change the path in the future, but let the name stay the same, we may change it however we like without affecting other places in our code. Also, it allows us to pass params much easily 😄